### ISSUE TITLE
We're mass-violating PEP8 for our Docopts comments!

### ISSUE THREAD

**Issue by 5310** [Hashfyre/pyzure-cli#16]
If PEP8 seems unreasonable, consider that these strings will also get shown on terminals, and hence they really ought to fit in 80 chars.

**Comment by 5310** [comment]
That said, Docopts also needs indented newlines for the parsing to work, which is a clause from against PEP8 we're already disregarding.

Maybe we should just disable PEP8 for docstrings?

But I Also want the docs to fit within 80 characters...

**Comment by Hashfyre** [comment]
Need further clarification with violation example, and exception to be made.

**Comment by 5310** [comment]
Install [`linter-pep8`](https://atom.io/packages/linter-pep8). (May seem snarky, but it's more efficient.)

**Comment by Hashfyre** [comment]
Installed, will update each defaulter file as we go.

**Comment by 5310** [comment]
But that's not the answer though. Docopts whitespacing goes against the strict Pep8 whitespacing rules, which don't make any sense when applied to docstrings anyway!

**Comment by 5310** [comment]
We've since discovered how Docopts needs an indented empty line right after the options explanation list, or else it breaks. So the `W293` warning (that atom-linter-pep8` unfairly raises as an error instead) marked within the docstrings has to stay.

We've added `W293` as an ignored error in `atom-linter-pep8`. 

But that is the only error I propose we ignore. We should not use docstring lines longer than 80 characters, as they'd be shown in terminals.
